index.ts

```diff
@@ -16,6 +16,13 @@ const bot = new Discord.Client({ intents: [Intents.FLAGS.GUILD_MESSAGES, Intents
 
 const Commands: Discord.Collection<string, Command> = new Discord.Collection();
 
+enum LOGLEVELS {
```

Comment:
Convention is order numerically. Additionally, should increment by 1 unless there is a specific reason why 2 is chosen.
```
ERROR = 1,
INFO = 2,
DEBUG = 3
```